trio/_core/_asyncgens.py

```diff
@@ -1,14 +1,21 @@
+from __future__ import annotations
+
 import logging
 import sys
 import warnings
 import weakref
+from collections.abc import AsyncGenerator
+from typing import TYPE_CHECKING, Any
 
 import attr
 
 from .. import _core
 from .._util import name_asyncgen
 from . import _run
 
+if TYPE_CHECKING:
+    from ._run import Runner
```

Comment:
We've imported `_run` already above, so you could just skip this import and use `_run.Runner` below.

Reply:
Changed in 0205c8f